core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -432,7 +438,10 @@ private void dominate(Competitor dominantTeam, Duration dominantTime) {
         capture(dominantTime);
       } else if (dominantTeam != null) {
         // non-capturing team is dominate, so regress capturing team's progress
-        recover(dominantTeam, dominantTime);
+        recover(dominantTeam, dominantTime, contested);
```

Comment:
Here is the other use for recover, here we do know for sure it serves no purpose